crates/cast/tests/e2e/declare.rs

```diff
@@ -173,13 +172,9 @@ fn scarb_no_artifacts(contract_path: &str, accounts_file_path: &str) {
     let snapbox = Command::new(cargo_bin!("sncast"))
         .current_dir(CONTRACTS_DIR.to_string() + contract_path)
         .args(args);
-    let assert = snapbox.assert().failure();
-    let stderr_output = String::from_utf8(assert.get_output().stderr.clone()).unwrap();
-
-    assert!(
-        stderr_output.contains(
-            "is set to 'true' under your [[target.starknet-contract]] field in Scarb.toml"
-        ),
-        "Expected error message not found in stderr: {stderr_output}",
-    );
+
+    snapbox.assert().failure().stderr_matches(indoc! {r#"
+        [..]is set to 'true' under your [[target.starknet-contract]] field in Scarb.toml[..]
```

Comment:
should this also include an information about command, to be in line with other outputs?